autoinstall/Pardus/preseed.cfg

```diff
@@ -227,17 +221,14 @@ d-i preseed/late_command string \
         then\
             echo "Add offical repo ..." > /target/dev/ttyS0;\
             codename=$(cat /target/etc/os-release | grep PARDUS_CODENAME | cut -d= -f2);\
-            chroot /target add-apt-repository "deb http://depo.pardus.org.tr/pardus $codename man contrib non-free";\
-            chroot /target add-apt-repository "deb http://depo.pardus.org.tr/guvenlik $codename man contrib non-free";\
+            echo "deb http://depo.pardus.org.tr/pardus $codename main contrib non-free" >> /target/etc/apt/sources.list;\
+            echo "deb http://depo.pardus.org.tr/guvenlik $codename main contrib non-free" >> /target/etc/apt/sources.list;\
             cat /target/etc/apt/sources.list > /target/dev/ttyS0;\
     fi; \
     echo "Update repository ..." > /target/dev/ttyS0; \
     chroot /target apt-get update -y > /dev/ttyS0; \
-    if [ ! -f "/target/etc/ssh/sshd_config" ];\
-        then\
-            echo "Install openssh-server ..." > /target/dev/ttyS0;\
-            chroot /target apt-get install -y openssh-server > /dev/ttyS0;\
-    fi; \
+    echo "Install openssh-server ..." > /target/dev/ttyS0; \
```

Comment:
Thanks

No necessory to check openssh-server is installed or not.
From the test result, "chroot /target apt-get install -y --no-upgrade openssh-server > /dev/ttyS0" works well for pardus 21.2 server and 21.5 XFCE.  
1) For pardus 21.2 server,  it prompts the openssh-servers is already installed and won't install it again when check the serial output
2) For pardus 21.4 XFCE, it will install openssh-server due to no openssh-server is already installed.


Thanks.

Reply:
Got it. Thanks!